src/notes/utils.ts

```diff
@@ -14,3 +17,11 @@ export function generatePublicId(): string {
   const randomId = randomBytes(16);
   return base32Encode(randomId, 'Crockford').toLowerCase();
 }
+
+/**
+ * Extract the primary alias from a aliases of a note
+ * @param {Note} note - the note from which the primary alias should be extracted
+ */
+export function getPrimaryAlias(note: Note): string {
+  return note.aliases.filter((alias: Alias) => alias.primary)[0].name;
```

Comment:
What happens if there is no primary alias?

Reply:
This should not happen, as if there are aliases one of them should be the primaryAlias. If there are actually no aliases `filter` should return `undefined`, which probably should be part of the return type.